Volume/Models/Flyer.swift

```diff
@@ -40,18 +41,78 @@ struct Flyer: Hashable, Identifiable {
+        ,
+        Flyer(date: DateInterval(start: retrieveDate(str: "Feb 3 23 6:00 PM"), end: retrieveDate(str: "Feb 3 23 7:30 PM")), description: "", id: "33", imageURL: "https://scontent-lga3-1.cdninstagram.com/v/t51.2885-15/328689714_551278403617394_2843396773937333049_n.jpg?stp=dst-jpg_e35&_nc_ht=scontent-lga3-1.cdninstagram.com&_nc_cat=102&_nc_ohc=ErFBmvWOO8wAX-CpPDZ&edm=ALQROFkBAAAA&ccb=7-5&ig_cache_key=MzAzMDEzOTgwMTE0MDE3NjU0MA%3D%3D.2-ccb7-5&oh=00_AfCEJks76kbRTLYS9vwucUIsQj4kqLr4-03yYGmbWxSyHg&oe=643AEEA4&_nc_sid=30a2ef", isFiltered: false, location: "WSH Memorial Room", organizations: [csa], organizationSlugs: ["csa"], pageURL: "https://www.instagram.com/p/CoNOBhCu6ac/", title: "Lunar New Year")
+        ,
+        Flyer(date: DateInterval(start: retrieveDate(str: "Apr 15 23 6:30 PM"), end: retrieveDate(str: "Apr 15 23 8:30 PM")), description: "", id: "34", imageURL: "https://scontent-lga3-1.cdninstagram.com/v/t51.2885-15/339137441_6237364136358386_7538052887918916053_n.jpg?stp=dst-jpg_e35&_nc_ht=scontent-lga3-1.cdninstagram.com&_nc_cat=102&_nc_ohc=VG5OieZMNqIAX_ZBGUp&edm=ACWDqb8BAAAA&ccb=7-5&ig_cache_key=MzA3MjQ4MzU1MzE3NDQ0NjUxNg%3D%3D.2-ccb7-5&oh=00_AfA4nSp68S_LTY1qH8BdNtkkEuCt-xKBhKaFDT06jUUB9A&oe=6439FD1F&_nc_sid=1527a3", isFiltered: false, location: "Barton Hall", organizations: [bhangra], organizationSlugs: ["bhangra"], pageURL: "https://www.instagram.com/p/Cqjp4NNMWG0/", title: "Pao XX")
+        ,
+        Flyer(date: DateInterval(start: retrieveDate(str: "Apr 14 23 4:30 PM"), end: retrieveDate(str: "Apr 14 23 6:00 PM")), description: "", id: "35", imageURL: "https://scontent-iad3-1.cdninstagram.com/v/t51.2885-15/340518148_236426522260137_7710221413690973548_n.jpg?stp=dst-jpg_e35&_nc_ht=scontent-iad3-1.cdninstagram.com&_nc_cat=109&_nc_ohc=gyqy0mPQIvsAX_kkRT7&edm=ACWDqb8BAAAA&ccb=7-5&ig_cache_key=MzA3ODY5NDMyODU0MDYyNzQwMw%3D%3D.2-ccb7-5&oh=00_AfAC9LVstQMMlurmTsq6_gMnYlsSdrNcZymLQKpdLBgaiQ&oe=643B27D1&_nc_sid=1527a3", isFiltered: false, location: "Hollister B14", organizations: [appdev], organizationSlugs: ["appdev"], pageURL: "https://www.instagram.com/p/Cq5uC4grIHL/", title: "Coffee & Create | Networking")
+        ,
+        Flyer(date: DateInterval(start: retrieveDate(str: "Mar 23 23 4:30 PM"), end: retrieveDate(str: "Mar 23 23 6:00 PM")), description: "", id: "36", imageURL: "https://scontent-iad3-1.cdninstagram.com/v/t51.2885-15/336290713_615486543771645_7718636499482251480_n.webp?stp=dst-jpg_e35&_nc_ht=scontent-iad3-1.cdninstagram.com&_nc_cat=109&_nc_ohc=-MJLkKT22rAAX8q-e49&edm=ACWDqb8BAAAA&ccb=7-5&ig_cache_key=MzA2MDU0MTgzNzExMDk4ODE1Nw%3D%3D.2-ccb7-5&oh=00_AfDzkS9Mzi4MPTM44AYK4WbWA4OtP-wtqPvuHGtSU3vmzQ&oe=6439B23C&_nc_sid=1527a3", isFiltered: false, location: "Olin Hall 255", organizations: [appdev], organizationSlugs: ["appdev"], pageURL: "https://www.instagram.com/p/Cp5OpXdOM19/", title: "Coffee & Create | Animation")
```

Comment:
i suggest you create a function to retrieve the `slug` fields from `organizations` and return an array for `organizationSlugs` to make this code more robust once the number of flyers starts increasing

Reply:
All of the logic I have right now will be removed once backend is done. I am forced to do this because we don't have much time.